src/modules/SystemSettings.rb

```diff
@@ -15,210 +15,304 @@ module Yast
-        SCR.Execute(
-          path(".target.bash"),
-          Builtins.sformat("echo '%1' > /proc/sys/kernel/sysrq", @ENABLE_SYSRQ)
-        )
-      else
-        Builtins.y2warning(
-          "Not activating invalid ENABLE_SYSRQ value: %1",
-          @ENABLE_SYSRQ
-        )
+    # Set SysRq keys status
+    #
+    # @param value [Boolean] true to enable them; false to disable
+    def SetSysRqKeysEnabled(value)
+      if value.nil?
+        log.warn("enable_sysrq should be 'true' or 'false'")
+        return
+      end
+
+      value_string = value ? "1" : "0"
+      if value_string != enable_sysrq
+        @modified = true
+        self.enable_sysrq = value_string
       end
 
-      if @elevator != nil
-        Yast.import "Bootloader"
-        new_elevator = @elevator == "" ? :missing : @elevator
+      nil
+    end
 
-        Builtins.y2milestone("Activating scheduler: %1", new_elevator)
-        # set the scheduler
-        Bootloader.modify_kernel_params("elevator" => new_elevator)
+    publish function: :GetPossibleElevatorValues, type: "list <string> ()"
+    publish function: :Modified, type: "boolean ()"
+    publish function: :Read, type: "boolean ()"
+    publish function: :Activate, type: "boolean ()"
+    publish function: :Write, type: "boolean ()"
+    publish function: :GetIOScheduler, type: "string ()"
+    publish function: :SetIOScheduler, type: "void (string)"
+    publish function: :GetSysRqKeysEnabled, type: "boolean ()"
+    publish function: :SetSysRqKeysEnabled, type: "void (boolean)"
+
+  protected
+
+    # Return configuration for SysRq keys
+    #
+    # This is the value that will be written when #Write is called. If you want tools
+    # get the real value from the system, check #kernel_sysrq and #sysctl_sysrq.
+    #
+    # @return [String] Configuration value; returns an empty string if it's not set.
+    #
+    # @see Write
+    # @see kernel_sysrq
+    # @see sysctl_sysrq
+    attr_reader :enable_sysrq
+
+    # Set sysctl configuration value for SysRq keys
+    #
+    # The value is not written until #Write is called.
+    #
+    # @param value [String] Configuration value
+    #
+    # @see Write
+    def enable_sysrq=(value)
+      log.info("SysRq was set to #{value}")
+      @enable_sysrq = value
+    end
 
-        # activate the scheduler for all disk devices
-        if new_elevator != :missing
-          Dir["/sys/block/*/queue/scheduler"].each do |f|
-            log.info "Activating scheduler '#{new_elevator}' for device #{f}"
-            File.write(f, new_elevator)
-          end
-        end
-      end
+    KERNEL_SYSRQ_FILE = "/proc/sys/kernel/sysrq".freeze
+
+    # Return kernel configuration value for SysRq keys
+    #
+    # The value is read from /proc/sys/kernel/sysrq
+    #
+    # @return [String] Configuration value; returns an empty string if it's not set.
+    def kernel_sysrq
+      return @kernel_sysrq unless @kernel_sysrq.nil?
```

Comment:
kernel_sysrq is string, so simple `return @kernel_sysrq if @kernel_sysrq` is enough